### PR TITLE
Qubes OS Summit: History from organizer's perspective

### PR DESCRIPTION
**Pull request by miczyg1**
Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

### CODE REVIEW

**Comment by miczyg1** [comment]
@pietrushnic @marmarek please review

File: 2022-08-07-qubes-os-summit-history.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+firmware and computer architecture, we were delighted to meet him in person in
+our office in Gdańsk, Poland. The time has come when the idea of a short event
+focused on firmware impact on Qubes OS security has been born. We asked Marek
+if he has some spare time to discuss firmware impacts on Qubes OS security and
+he liked the idea. The event has been called humorously a "minisummit". That is
+how the first Qubes OS summit started, one day, in a small conference room,
+with a tiny group of people passionate about firmware and system security,
+nothing outrageous (in the meaning of scale). One may still read what topics
+have been discussed
+[here](https://blog.3mdeb.com/2019/2019-08-07-qubes-os-and-3mdeb-minisummit/).
```
Comment (pietrushnic) [inline 2022-08-07-qubes-os-summit-history.md:25]: @miczyg1 very well written IMHO.  Please add topic examples shortly here and in other places, so one without visit could realize what we talking about during such events.

File: 2022-08-07-qubes-os-summit-history.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+In August 2021 we held yet another Qubes OS Summit together with the Qubes OS
+team, still virtually, unfortunately. However, this year brought more speakers
+from other companies and projects, which we were very happy about. There were 2
+sessions almost 4 hours each and one may watch the talks at
+[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIoLQzpYeZnkupURheXky6r).
+
+# The future of Qubes OSe Summits
+
+Finally, the time has come when we may meet together in person, after two
```
Comment (pietrushnic) [inline 2022-08-07-qubes-os-summit-history.md:56]: two? not three? in person event was in 2019, we saw Marek during FOSDEM 2020 mabe, so at least `over two years`. When I'm counting my personal attendence last time it was Embedded World in February 2020.

File: 2022-08-07-qubes-os-summit-history.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+modern hardware. [Here](https://cfp.3mdeb.com/qubes-os-summit-2022/schedule/#)
+you may find the full schedule. Among others we welcome the following speakers:
+
+* Wessel Klein Snakenborg from Novacustom with a story of open-source on modern
+  laptops as a candidate to Qubes OS certification
+* Brent Cowing from Protectli talking about security of his small form factor
+  computers also running open-source firmware
+* Jan Suhr from Nitrokey talking about how enterprise requirements tied to
+  Windows can be met with Qubes OS
+* Arthur Rasmusson from OpenMdev project about GPU virtualization and its
```
Comment (pietrushnic) [inline 2022-08-07-qubes-os-summit-history.md:76]: We probably should say Arc Compute here

File: 2022-08-07-qubes-os-summit-history.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  possible benefits to QUbes oS
```
Comment (pietrushnic) [inline 2022-08-07-qubes-os-summit-history.md:77]:
```suggestion
  possible benefits to Qubes OS
```

File: 2022-08-07-qubes-os-summit-history.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+* Puck Meerburg from Spectrum OS telling about the power of Wayland for GUI
+  isolation and how it benefits systems like Qubes OS or Spectrum OS
+* And and everyone from 3mdeb, Qubes OS team and others
+
+We hope that the event and community around it and the Qubes OS project will
+grow in the future, bringing benefits and collaboration for everybody.
```
Comment (pietrushnic) [inline 2022-08-07-qubes-os-summit-history.md:83]:
```suggestion
grow in the future, bringing reasonable trustworthiness and collaboration for everybody.
```

?

File: 2022-08-07-qubes-os-summit-history.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+Definitely, 3mdeb will try its best to co-organize the event in the upcoming
+years.
+
+# Summary
+
+By a mere coincidence, meeting Marek Marczykowski-Górecki gave birth to a
+fascinating series of Qubes OS Summits and led us to this day.
+
+We (3mdeb) are very proud to be the co-organizers of the 4th edition of the
```
Comment (pietrushnic) [inline 2022-08-07-qubes-os-summit-history.md:92]: Maybe rephrase that in way as follows:

Together with InvisibleThingsLab we (3mdeb) are very proud ....

File: 2022-08-07-qubes-os-summit-history.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+from other companies and projects, which we were very happy about. There were 2
+sessions almost 4 hours each and one may watch the talks at
+[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIoLQzpYeZnkupURheXky6r).
+
+# The future of Qubes OSe Summits
+
+Finally, the time has come when we may meet together in person, after two
+years! The Qubes OS Summit 2022 is fast approaching, with only a few days left
+before the start of the event in Berlin. As it is no longer held only virtually
+(but still possible to attend remotely for those who can be with us in person)
```
Comment (mfc) [inline 2022-08-07-qubes-os-summit-history.md:59]: can -> can't

File: 2022-08-07-qubes-os-summit-history.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Qubes OS Summit: History from organizer's perspective"
+categories: talks
+---
+
+The next [Qubes OS Summit 2022](https://www.qubes-os.org/news/2022/07/29/qubes-os-summit-2022/) edition is upcoming. This year it will be held live in Berlin from September 9th to 11th. Don't miss the event and more importantly how it started. In the article [Michał Żygowski from 3mdeb](https://blog.3mdeb.com/authors/michal-zygowski/) (Qubes OS Summit co-organizer) will describe the history and organizer's perspective of the event.
```
Comment (mfc) [inline 2022-08-07-qubes-os-summit-history.md:7]: probably worth mentioning here that it will be live-streamed and accessible remotely

Comment (mfc) [reply 2022-08-07-qubes-os-summit-history.md:7]: and linking to the actual summit website with more info

File: 2022-08-07-qubes-os-summit-history.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Qubes OS Summit: History from organizer's perspective"
+categories: talks
```
Comment (miczyg1) [inline 2022-08-07-qubes-os-summit-history.md:4]: @mfc I'm wondering if I chose the category correctly. Wouldn't the article category be more appropriate?

File: 2022-08-07-qubes-os-summit-history.md (hunk 8 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md]: Done

File: 2022-08-07-qubes-os-summit-history.md (hunk 1 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md:25]: Done for all previosu editions.

File: 2022-08-07-qubes-os-summit-history.md (hunk 2 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md]: Yes, three. Fixed.

File: 2022-08-07-qubes-os-summit-history.md (hunk 7 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md]: Fixed

File: 2022-08-07-qubes-os-summit-history.md (hunk 3 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md]: Fixed

File: 2022-08-07-qubes-os-summit-history.md (hunk 4 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md]: Fixed

File: 2022-08-07-qubes-os-summit-history.md (hunk 5 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md]: Applied

File: 2022-08-07-qubes-os-summit-history.md (hunk 6 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md]: Done

File: 2022-08-07-qubes-os-summit-history.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+challenge for us because we have organized such a virtual event for the first
+time. Fortunately, everything went smoothly and finished with success after 4
+sessions 2.5 hours long.
+
+In August 2021 we held yet another Qubes OS Summit together with the Qubes OS
+team, still virtually, unfortunately. However, this year brought more speakers
+from other companies and projects, which we were very happy about. There were 2
+sessions almost 4 hours each and one may watch the talks at
+[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIoLQzpYeZnkupURheXky6r).
+This time the 3mdeb's stage belonged to Piotr Król (CEO of 3mdeb) who presented
```
Comment (marmarek) [inline 2022-08-07-qubes-os-summit-history.md:59]:
```suggestion
This time, the 3mdeb's stage belonged to Piotr Król (CEO of 3mdeb) who presented
```

File: 2022-08-07-qubes-os-summit-history.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+  laptops as a candidate to Qubes OS certification
+* Brent Cowing from Protectli talking about security of his small form factor
+  computers also running open-source firmware
+* Jan Suhr from Nitrokey talking about how enterprise requirements tied to
+  Windows can be met with Qubes OS
+* Arthur Rasmusson from Arc Compute talking about GPU virtualization and its
+  possible benefits to Qubes oS
+* Puck Meerburg from Spectrum OS telling about the power of Wayland for GUI
+  isolation and how it benefits systems like Qubes OS or Spectrum OS
+* And and everyone from 3mdeb, Qubes OS team and others
```
Comment (marmarek) [inline 2022-08-07-qubes-os-summit-history.md:92]: And and

**Review by marmarek — CHANGES_REQUESTED** [review]
Few minor comments, otherwise looks good

**Review by pietrushnic — APPROVED** [review]

File: 2022-08-07-qubes-os-summit-history.md (hunk 11 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md:92]:
```suggestion
* And everyone from 3mdeb, Qubes OS team, and others
```

**Comment by miczyg1** [comment]
> Few minor comments, otherwise looks good

Applied through suggestions, thank you.

File: 2022-08-07-qubes-os-summit-history.md (hunk 9 above)
Comment (andrewdavidwong) [reply 2022-08-07-qubes-os-summit-history.md:4]: You can use both:

```
categories:
- talks
- articles
```

File: 2022-08-07-qubes-os-summit-history.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Qubes OS Summit: History from organizer's perspective"
+categories: talks
```
Comment (andrewdavidwong) [inline 2022-08-07-qubes-os-summit-history.md:4]: Please add an "author" field, like this:

```
author: <name>
```

`<name>` can be `3mdeb`, a comma-separated list of your personal names, or whatever else is appropriate.

File: 2022-08-07-qubes-os-summit-history.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+---
+
+The next [Qubes OS Summit 2022](https://www.qubes-os.org/news/2022/07/29/qubes-os-summit-2022/) edition is upcoming. This year it will be held in Berlin from September 9th to 11th in hybrid format, in person and live-streamed for remote access. More details [here](https://qubesos.3mdeb.com/). Don't miss the event and more importantly how it started. In the article [Michał Żygowski from 3mdeb](https://blog.3mdeb.com/authors/michal-zygowski/) (Qubes OS Summit co-organizer) will describe the history and organizer's perspective of the event.
```
Comment (andrewdavidwong) [inline 2022-08-07-qubes-os-summit-history.md:7]: Before this is published, I will be adding a short editor's note at the very top that incorporates some of this information so that readers understand that this is a guest post ([example](https://www.qubes-os.org/news/2022/04/10/windows-integration-by-tabit-pro/), so there is no need to self-preface. You can instead simply start at the beginning of your topic, if you prefer.

File: 2022-08-07-qubes-os-summit-history.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+from other companies and projects, which we were very happy about. There were 2
+sessions almost 4 hours each and one may watch the talks at
+[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIoLQzpYeZnkupURheXky6r).
+This time, the 3mdeb's stage belonged to Piotr Król (CEO of 3mdeb) who presented
+his adventures using Qubes OS as an everyday device, talking about USB camera
+and cryptocurrency wallets. Of course firmware-related topics could not be
+missing and Piotr also showed continuation of his previous year's efforts of
+securing the VMs with SRTM and Secure Boot.
+
+# The future of Qubes OSe Summits
```
Comment (andrewdavidwong) [inline 2022-08-07-qubes-os-summit-history.md:65]: Typo: `OSe`

File: 2022-08-07-qubes-os-summit-history.md (hunk 12 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md:4]:
```suggestion
categories: talks
author: Michał Żygowski
```

File: 2022-08-07-qubes-os-summit-history.md (hunk 9 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md:4]:
```suggestion
categories:
- talks
- articles
```

File: 2022-08-07-qubes-os-summit-history.md (hunk 13 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md]: Moved to a separate section.

File: 2022-08-07-qubes-os-summit-history.md (hunk 12 above)
Comment (miczyg1) [reply 2022-08-07-qubes-os-summit-history.md]: Added

**Review by marmarek — APPROVED** [review]

**Comment by andrewdavidwong** [comment]
Published. Thanks, @miczyg1! :)
